src/packet/packet_structure.rs

```diff
@@ -32,43 +52,198 @@ impl Packet {
     ///
     /// Basically just bare UDP, free to be dropped, used for very unnecessary data, great for 'general' position updates.
```

Comment:
```suggestion
    /// Basically just bare UDP. The packet may or may not be delivered.
```

Reply:
I changed this because I don't think we should suggest use cases in these comments, but instead in the docs.